pkg/install/helm/settings.go

```diff
@@ -97,23 +96,24 @@ func (n *NormalisedEnvSettings) setupEnvSettings(ctx context.Context, cmd *cobra
 	{
 		// Add a PreRun hook to set the debug value to true if the log level is
 		// >= 3.
-		existingPreRun := cmd.PreRun
-		cmd.PreRun = func(cmd *cobra.Command, args []string) {
+		existingPreRun := cmd.PersistentPreRunE
+		cmd.PersistentPreRunE = func(cmd *cobra.Command, args []string) error {
 			if n.logger.V(debugLogLevel).Enabled() {
 				n.EnvSettings.Debug = true
 			}
 
 			if existingPreRun != nil {
-				existingPreRun(cmd, args)
+				return existingPreRun(cmd, args)
 			}
+			return nil
 		}
 	}
 }
 
 func (n *NormalisedEnvSettings) InitActionConfiguration() error {
 	return n.ActionConfiguration.Init(
 		n.Factory.RESTClientGetter,
-		n.EnvSettings.Namespace(),
+		n.Factory.Namespace,
```

Comment:
The n.Factory.Namespace seems like a better source-of-truth to get the namespace.